components/nifosg/particle.hpp

```diff
@@ -30,11 +30,13 @@ namespace NifOsg
         ParticleSystem();
         ParticleSystem(const ParticleSystem& copy, const osg::CopyOp& copyop);
 
-        META_Object(NifOsg, NifOsg::ParticleSystem)
```

Comment:
Yes please take these if you're willing.   I've tried to change as little as possible in base OpenMW as possible because I wasn't sure if you'd even want these changes.